eda/README.md

```diff
@@ -0,0 +1,36 @@
+# EDA
+
```

Comment:
no need for a readme. please remove this file and put the notebook in pre-processing folder.